web-client/src/translations/translations.json

```diff
@@ -29,6 +38,15 @@
         "pleaseWaitLabel": "Aguarde por favor...",
         "tip_title": "Lembrete",
         "tip_info": "Assim que um voluntário oferecer ajuda, deve aceitar a sua ajuda para que ele consiga ver os seus contactos"
+      },
+      "roleinfo": {
+        "title": "Reach4Help",
```

Comment:
This could probably go into `common` as we're probably using this elsewhere :)

Reply:
Agree.